projects/media-viewer/src/lib/media-viewer.component.ts

```diff
@@ -80,6 +85,9 @@ export class MediaViewerComponent implements OnChanges, OnDestroy, AfterContentI
       if (this.enableAnnotations) {
         this.annotationSet = this.api.getOrCreateAnnotationSet(this.url);
       }
+      if (this.contentType === 'image') {
+        this.documentTitle = null;
```

Comment:
why is this set to null for images?

Reply:
we want to display document title in comment summary for pdfs and just 'Comments Summary' for images

Reply:
im okay with this.